app/components/createContainer/createcontainer.html

```diff
@@ -350,6 +350,22 @@
                 </div>
               </div>
               <!-- !domainname -->
+              <!-- ipv4-input -->
+              <div class="form-group">
+                <label for="container_ipv4" class="col-sm-2 col-lg-1 control-label text-left">IPv4 Address</label>
+                <div class="col-sm-9">
+                  <input type="text" class="form-control" ng-model="formValues.IPv4" id="container_ipv4" placeholder="(Optional) e.g. xxx.xxx.xxx.xxx">
+                </div>
+              </div>
+              <!-- !ipv4-input -->
+              <!-- ipv6-input -->
+              <div class="form-group">
+                <label for="container_ipv6" class="col-sm-2 col-lg-1 control-label text-left">IPv6 Address</label>
+                <div class="col-sm-9">
+                  <input type="text" class="form-control" ng-model="formValues.IPv6" id="container_ipv6" placeholder="(Optional) e.g. xxxx:xxxx::xx">
```

Comment:
Could you change the placeholder value to `e.g. a:b:c:d::1234` instead of `(Optional) e.g. xxxx:xxxx::xx` ?